tests/test_utils.py

```diff
@@ -832,3 +927,69 @@ def test_validate_email_address(email_address, is_valid):
 )
 def test_shorten_str(original, shorten_by, shortened):
     assert shorten_str(original, shorten_by) == shortened
+
+
+@pytest.mark.asyncio
+async def test_get_event_loop_uvloop():
+    with patch("asyncio.set_event_loop_policy") as set_event_loop_policy_mock:
+        get_event_loop(uvloop=True)
+        set_event_loop_policy_mock.assert_called()
+
+
+@pytest.mark.asyncio
+async def test_get_event_loop_uvloop_when_exception():
+    with patch("asyncio.set_event_loop_policy", side_effect=Exception("welp")):
+        # Event if there's an exception, the loop is created
+        get_event_loop(uvloop=True)
+        assert True
```

Comment:
do you need to close it?

Reply:
I'm quite confused by this code TBH, in the two tests above I did not need to close the loop, in the tests below I did, most likely because there's current event loop running for test (it's marked as `async`). 

Inside of the test I create a new event loop, and then I need to close it. If I would not create a new one, there would be no need to close anything.

However, I close 2 loops and this is something I'm confused about, I'll do a bit more research here.